common/local-env/rabbit/dockerfile

```
@@ -0,0 +1,2 @@
+FROM rabbitmq:3.7-management
+RUN rabbitmq-plugins enable --offline rabbitmq_amqp1_0
```

Not sure if this will add 2 newlines or one, but:
```suggestion
RUN rabbitmq-plugins enable --offline rabbitmq_amqp1_0

```

---

done locally.